modules/apps/portal-odata/portal-odata-impl/src/main/java/com/liferay/portal/odata/internal/filter/ExpressionVisitorImpl.java

```diff
@@ -576,6 +635,23 @@ private Object _normalizeDateLiteral(String literal) {
 		}
 	}
 
+	private Object _normalizeDurationLiteral(String literal) {
+		literal = StringUtil.removeSubstring(literal, "duration");
+
+		literal = StringUtil.unquote(literal);
```

Comment:
It is not possible to apss  StringUtil.removeSubstring(literal, "duration") as a parameter to StringUtil.unquote(literal)?

Reply:
yes , I can do it.